test/utility/create-person.ts

```diff
@@ -43,7 +43,7 @@ export async function createPerson(
   expect(actual).toBeTruthy();
 
   expect(isValidId(actual.id)).toBe(true);
-  expect(actual.email.value).toBe(person.email.toLowerCase());
+  expect(actual.email.value).toBe(person.email?.toLowerCase());
```

Comment:
```suggestion
```

There shouldn't really be any assertions here. The id is really just a sanity check that the gql call was successful. 

If we want to assert the email then that should be directly in a test case.